### DISCUSSION TITLE
Herd and HELO

### DISCUSSION THREAD

**Discussion by daugaard47** [beyondcode/herd-community#1202] [Q&A]
Curious, I have HELO pro and the Pro version of Herd, but noticed Herd mail is missing the sharing feature.
1. In future Herd versions do you plan to flesh out herd mail more like HELO?
2. In future Herd (free) versions would you consider allowing previous HELO users access to Herd mail pro by adding their HELO license?

**Comment by woodymendoza** [comment]
> **1. In future Herd versions do you plan to flesh out herd mail more like HELO?**
^
